config.ini

```diff
@@ -3,7 +3,7 @@
 # This should almost always be set to False.
```

Comment:
remove this comment if the default is True

Reply:
my bad, committed a change I was using for testing

Reply:
fixed in #91